src/js/views/maps/ExpansionPanelView.js

```diff
@@ -70,16 +71,30 @@ define(
          * @property {boolean} startOpen Whether the panel should be expanded by
          * default. 
          */
-        initialize({ title, contentViewInstance, icon, panelsModel, startOpen }) {
-          this.templateVars.title = title;
-          this.templateVars.icon = icon;
+        initialize({ title, contentViewInstance, icon, panelsModel, startOpen, isSvgIcon, }) {
+          this.templateVars = {
+            ...this.templateVars,
+            title: title,
+            icon: icon,
+            isSvgIcon: isSvgIcon,
+          };
           this.contentViewInstance = contentViewInstance;
           this.panelsModel = panelsModel;
           this.startOpen = !!startOpen;
+          this.icon = icon;
```

Comment:
can we just reference this.templateVars.icon instead of creating another variable?

Reply:
Yep, done